### PR TITLE
docs: video covering adding code servers

### PR DESCRIPTION
**Pull request by misterbisson**

### CODE REVIEW

File: content/how-to/code-servers/add-code-servers/github-cloud/_index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "Adding a GitHub Cloud code server"
 aliases:
   - /how-to/add-code-servers/github-cloud
 description: "How to add a GitHub cloud code server to your tenant"
-lead: "How to add a GitHub Cloud code server to your tenant"
+lead: ""
```
Comment (misterbisson) [inline content/how-to/code-servers/add-code-servers/github-cloud/_index.md:6]: Unrelated cleanup: removing redundant text.

**Review by srinivasakartheek — APPROVED** [review]
